idc requires <qt_windows.h>, QProcess, and QLibraryInfo, none of which
are available with libQt5Bootstrap.

--- origsrc/qtactiveqt-opensource-src-5.2.0/src/src.pro	2013-12-08 11:09:36.000000000 -0600
+++ src/qtactiveqt-opensource-src-5.2.0/src/src.pro	2014-01-05 15:09:09.920370500 -0600
@@ -1,4 +1,4 @@
 TEMPLATE = subdirs
 SUBDIRS = activeqt
-win32: SUBDIRS += tools
+!cross_compile:win32: SUBDIRS += tools
 CONFIG += ordered
